site/docs/guides/spark-s3.md

```diff
@@ -0,0 +1,91 @@
+# Accessing data in S3 with Spark
+
+In this guide we walk through the process of configuring an [Apache Spark](https://spark.apache.org/) session to work 
+with data files stored in Amazon [S3](https://aws.amazon.com/s3/) and version history in a local Nessie Server.
+
+Docker is used at the runtime environments for Nessie. Spark is assumed to be installed locally.
+
+## Setting up Nessie Server
+
+Start the Nessie server container from the `projectnessie/nessie` Docker image in default mode.
+
+```shell
+docker run -p 19120:19120 projectnessie/nessie:latest
+```
+
+Note: this example with run the Nessie Server using in-memory storage for table metadata. If/when the container
+is deleted Nessie data about table changes will be lost, yet the data files in S3 will remain.
```

Comment:
missing a comma here: ...is deleted, Nessie data about...

Reply:
ah... thx... to long away from school ;)